llpc/lower/llpcSpirvLowerRayTracing.cpp

```diff
@@ -1085,20 +1098,133 @@ Value *SpirvLowerRayTracing::processBuiltIn(unsigned builtInId, Instruction *ins
 // Create shader table variable
 //
 // @param tableKind : Kind of shader table variable to create
-GlobalVariable *SpirvLowerRayTracing::createShaderTableVariable(ShaderTable tableKind) {
+// =====================================================================================================================
+// Create shader table variable
+//
+// @param tableKind : Kind of shader table variable to create
+// @param bufferDesc : Dispatch ray buffer descriptor
+Value *SpirvLowerRayTracing::createShaderTableVariable(ShaderTable tableKind, Value *bufferDesc) {
```

Comment:
createShaderTableVariable -> loadShaderTableVariable

Also, remove the `bufferDesc` argument since it's taken from a member variable anyway.

Reply:
m_dispatchRaysInfoDesc is passed to the dialect function call -- OpReortHit/OpCallCallable then-> getShaderIdentifier() -> createShaderTableVariable() . so m_dispatchRaysInfoDesc is not accessable to these functions.